src/OpenLoco/OpenLoco.h

```diff
@@ -33,6 +33,14 @@ namespace OpenLoco
 
     extern const char version[];
 
+    namespace GameSpeed
```

Comment:
Can this become a strong enum?

Reply:
I originally had it as one, but had to keep casting to the underlying type everywhere and it was messy. Happy to change it back, but it just adds more code and IMO it didn't look at clean

Reply:
I would also suggest to use a strong enum. I don't see how it would need to use the underlying type often other than maybe increment/decrement the speed.

Reply:
Unless you feel strongly, I think the code is much cleaner with this as 'constants in a namespace' than an enum class, so I won't change it in this PR. @AaronVanGeffen do you have any thoughts on this?

Reply:
The point is having a strong type for this so we don't have to look at uint8_t and figure out what sort of data it expects.

Reply:
Thanks, I know what the point of a enum classes is, and as I mentioned above, I already implemented this change with game speed as an enum class. It wasn't clean. It added a bunch more code and overall the benefit of adding the enum class was negated by the messy code. Enum classes (and any other language feature) are simply a tool in your toolbox to consider, not something that must be used every time a potential use case pops up.

Reply:
Did you actually replace the uint8_t with the enum type? I don't exactly see what would be not clean using it.

Reply:
I did, but since it seems so controversial I'll redo it and you can see yourself

Reply:
Its not as bad as you make it to be imho. If you were to change the static variable type you would avoid another static_cast. As for the game command wrappers, they are temporary until the requirement to call it like this is gone so would be the static casts which for long term is probably desired.